contracts/test/integration/MultiSubnet.t.sol

```diff
@@ -607,7 +617,12 @@ contract MultiSubnetTest is Test, IntegrationTestBase {
         CheckpointingFacet checkpointer = CheckpointingFacet(address(gateway));
 
         BottomUpMsgBatch memory batch = getter.bottomUpMsgBatch(e);
-        require(batch.msgs.length == 1, "batch length incorrect");
+        console.log("batch length %d", batch.msgs.length);
+        require(batch.msgs.length > 0, "batch length incorrect");
+        if (batch.msgs.length == 2) {
+            printEnvelope(batch.msgs[0]);
+            printEnvelope(batch.msgs[1]);
+        }
```

Comment:
Relaxing the require here seems good. There is no reason why this must only handle a single message per bottom-up checkpoint.